src/js/utils/can-autoplay.js

```diff
@@ -29,6 +29,7 @@ const URL = (global => global.URL && global.URL.createObjectURL ?
     global.URL : global.webkitURL || global.mozURL)(window);
 
 let videoBlob;
+const Blob = window.Blob;
```

Comment:
i assume this is what the new lint rule is for? is it worth calling it something diff? are there risks with having a var the same name as a window var?

Reply:
Nothing wrong with shadowing global names. We should always get globals from the global object (`window` or `self`).